assets/js/translator.js

```diff
@@ -460,6 +460,11 @@ function populateTranslationList() {
         numDstCols = Math.ceil(dstLangs.length / TRANSLATION_LIST_ROWS) < (TRANSLATION_LIST_COLUMNS + 1)
             ? Math.ceil(dstLangs.length / TRANSLATION_LIST_ROWS)
             : TRANSLATION_LIST_COLUMNS;
+
+    var thresholdWindowWidth = 1000;
```

Comment:
Ohh yep! Um, so my approach was - 
1. Looking at the code snippet that decides the number of columns, I observed that the maximum number of columns that can come is 4 ( as we limit it to `TRANSLATION_LIST_COLUMNS` ).
2. Now, having less than 4 columns does not hurt in any window width as I checked. Thus, dynamically changing the number of columns to 3 ( ie decreasing by 1 ), under the condition that width < 1000 && number of columns is 4, will get the language list into 3 columns thus solving our problem.